lib/ferrum/contexts.rb

```diff
@@ -11,6 +11,7 @@ class Contexts
     def initialize(client)
       @contexts = Concurrent::Map.new
       @client = client
+      @default_context = create_default_context if @client.options.use_default_context
```

Comment:
One side effect of setting this in the constructor is that any existing tabs/targets in the browser's default context will be available as targets (e.g. `browser.default_context.targets` will return a nonzero number of targets). This felt right to me, because then it handles use cases like https://github.com/rubycdp/ferrum/issues/320 as well. Similar to the fork mentioned in https://github.com/rubycdp/ferrum/issues/320#issuecomment-1368368807 but in my opinion cleaner because it just gives you access to all the existing tabs to do with what you will, rather than somewhat arbitrarily picking the default (latest?) tab to be the default target.